### PR TITLE
Code reorganization and minor misc changes

### PR DESCRIPTION
**Pull request by toph-allen**
- Moved `warning_message()` and `info_message()` to their own `messages.R` file.
- Created `plumbertableau-package.R` so that `?plumbertableau` shows package-level help.
- Moved `parseQS()` and `combine_keys()` to `reroute.R`.
- Minor update to `README.md`
- Minor updates to test descriptions to name the functions they describe.

### CODE REVIEW

File: R/plumbertableau-package.R [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#' @keywords internal
+"_PACKAGE"
```
Comment (blairj09) [inline R/plumbertableau-package.R:2]: I'm not sure if I understand why we need this

Comment (toph-allen) [reply R/plumbertableau-package.R:2]: With this, you can type `?plumbertableau` at the R command and see package help. It's important for onboarding new users who might not know anything other than the package name.

Comment (toph-allen) [reply R/plumbertableau-package.R:2]: Specifically, this is what `?plumbertableau` looks like:
Screen Shot 2021-07-20 at 1 09 42 PM

Comment (blairj09) [reply R/plumbertableau-package.R:2]: TIL. Sounds great!

**Pull request merged**